### DISCUSSION TITLE
HERO11 Mini Labs release 2.10.70

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#225] [Announcements]
This is a nice update with more in the works.

The Release notes are updated here: https://gopro.github.io/labs/control/notes/

New firmware : https://gopro.github.io/labs/firmware/bin/LABS_MINI11_02_10_70.zip

Let me know you thoughts and experiences with this update.

**Comment by andrerav** [comment]
Fantastic work! Stoked to try out GCMP and the new color extensions when/if this hits the GP11!

**Reply by dnewman-gpsw** [reply]
Thanks. Yes, these features and more are coming to HERO11.  Labs release in month of so.  H11 and Mini releases are staggered for engineering and QA bandwidth.

**Comment by Shadetail** [comment]
> Added WIDE (flat color wider gamut, like wN with white balance controls), 2020 (color primaries), MTRX (color matrix) extensions.

I'll try to unpack this to confirm if I'm understanding it correctly. (edit: I was wrong, see comment below)

So the syntax would be:
!MWIDE=MTRX
!MWIDE=2020

I'm under the impression that white balance native changes the color primaries to those which are in some sense native to the sensor, and at the same time uses the native white balance of the sensor, presumably 6500K (D65).
As I understand the release notes, WIDE=MTRX is like using native white balance but without being locked out of having white balance controls, so that native primaries from native white balance mode are used, but white balance can still be set to auto or to a manual value. 

And setting it to WIDE=2020 would have the same effect of freeing up white balance controls for those of us who have been using native white balance to shoot HDR, but we will no longer need to use a lut to transform the color space from gopro native to rec.2020, since footage will now be in rec.2020 by default.

I went and [plotted the GoPro native color primaries against rec.2020](https://i.imgur.com/GvWE2no.png) and it turns out that in theory using 2020 should provide larger color gamut for HDR than using native, which is fantastic news if so, but should be tested experimentally, which I plan to do.

Is mention of "flat color" in the quote above meant to imply that WIDE only works with "flat color" setting?

([Source for native primaries and white point.](https://colour.readthedocs.io/en/v0.3.10/colour.models.rgb.dataset.gopro.html))

**Comment by dnewman-gpsw** [comment]
Sorry, those features are poorly documented, some weren't intended to be listed.

oMWIDE=1 enables a wide color gamut, sensor native, while still supporting white balance.

oM2020=1 it is such a crude hack, I was not intending to list it.  It does use 2020 color targets, but it still has limited gamut. So don't use this, as there is zero advantage resulting.

oMMTRX=( nine values of the RGB to YCrCb color matrix, all integers where 256 is 1.0). Again an incomplete hack, in theory to you use WIDE=1 and then use a custom color matrix to target 2020 correctly. Using a LUT would be easier.  In the meantime the only thing you can do are color effects, like making a B/W camera oMMTRX=50,175,25,0,0,0,0,0,0
![image](https://user-images.githubusercontent.com/10856084/216878258-16902b36-357d-4a89-9a4e-44fe17ee1a51.png)
